package.json

```diff
@@ -73,8 +69,27 @@
         "@babel/preset-env": "^7.24.3",
         "@babel/preset-react": "^7.24.1",
         "@babel/preset-typescript": "^7.24.1",
-        "@types/react": "^18.2.73",
-        "@types/react-dom": "^18.2.23",
+        "@types/core-js": "^2.5.8",
+        "@types/dagre": "^0.7.52",
+        "@types/eslint": "^7.29.0",
+        "@types/eslint-config-prettier": "^6.11.3",
+        "@types/identity-obj-proxy": "^3.0.2",
+        "@types/jest": "^29.5.12",
+        "@types/license-checker": "^25.0.6",
+        "@types/localized-countries": "^2.0.3",
+        "@types/lucene-escape-query": "^1.0.2",
+        "@types/node": "^18.19.29",
```

Comment:
Same

Reply:
I'm not sure for this one. Can you give more details?

Reply:
alright